server/src/models/staff/staff.dto.ts

```diff
@@ -47,14 +47,17 @@ export class UpdateStaffDto {
 
   @IsEmail()
   @IsNotEmpty()
+  @IsOptional()
```

Comment:
I don't like this approach, seems like we are cutting corners that could bite us back in the future.